{{cookiecutter.project_slug}}/src/navigator/mainNavigator.js

```diff
@@ -1,27 +1,74 @@
-import { createStackNavigator, createAppContainer } from "react-navigation";
+import React, { Component } from "react";
+
+import { createStackNavigator, createAppContainer, createDrawerNavigator } from "react-navigation";
 
 import SplashScreen from "../features/SplashScreen";
 
 {% if cookiecutter.has_login_blueprint %}import { EmailAuthNavigator } from "../features/EmailAuth/navigator";{% endif %}
 
+import DrawerMenu from '../components/DrawerMenu';
+import ProfileScreen from "../features/Profile/index";
+import { Dimensions } from 'react-native';
+
+const dm = Dimensions.get('screen');
+
 /**
  * new navigators can be imported here
  */
 
+const MainDrawerNavigator = createDrawerNavigator(
```

Comment:
This navigator should not be in the embed into mainNavigator, you should create a subnavigator inside your feature folder and import it here using conditional variables.

You can check EmailAuth profile as reference on how to implement it